internal/pkg/cli/svc_deploy.go

```diff
@@ -45,8 +45,10 @@ import (
 )
 
 const (
-	fmtForceUpdateSvcStart    = "Forcing an update for service %s from environment %s"
-	fmtForceUpdateSvcFailed   = "Failed to force an update for service %s from environment %s: %v.\n"
+	fmtForceUpdateSvcStart  = "Forcing an update for service %s from environment %s"
+	fmtForceUpdateSvcFailed = `Failed to force an update for service %s from environment %s: %v.
```

Comment:
nit: `svc status` probably wouldn't help much if the failure is due to a failed attempt at calling `UpdateService`, when tasks can't even be started. 

I feel like `svc status` is most useful in debugging when circuit breaker is triggered, where tasks are started and stopped over and over again.

Maybe we can advise users to use `svc status` only when it's a waiting error, or we can modify the log message accordingly? What do you think?

Reply:
Right right. Good call